docs/integrations/azure/locations.md

```diff
@@ -45,6 +67,53 @@ integrations:
         - personalAccessToken: ${PERSONAL_ACCESS_TOKEN}
 ```
 
+### Using a service principal with a managed identity to generate the client assertion
+
+Using a managed identity to generate a client assertion is an advanced scenario. It requires you to setup a federated credential for the app registration in Azure Entra ID.
+
+It is most useful when you want to [authenticate against an Azure DevOps organization in a different tenant](#authenticate-against-an-azure-devops-organization-in-a-different-tenant) than the managed identity itself. Otherwise a regular managed identity is probably a more suitable choice.
+
+#### Add a federated credential
+
+To be able to use a managed identity to generate a client assertion, you need to create a federated credential in Azure Entra ID. Follow these steps:
+
+1. Create an app registration in Entra ID (or use an existing one).
+2. Navigate to the `Certificates & secrets` tab for your app registration.
+3. Add a new federated credential using the `Customer managed keys` scenario.
+4. Select the managed identity you want to use to generate the client assertion.
+5. Enter the name and description.
+6. Click `Add`.
```

Comment:
For these steps can we uses plain quotes (") over backticks (`) so that these don't have code fences around them? This makes it easier to read.

Reply:
Done